src/utils/runConfigs.js

```diff
@@ -0,0 +1,287 @@
+export const LocalRun = {
+  type: 'LocalRun',
+  label: 'Local',
+  description: 'Run the flow in a local process on a local agent',
+  ref: 'https://docs.prefect.io/api/latest/run_configs.html#localrun',
+  icon: 'fad fa-laptop-house',
+  args: [
+    {
+      arg: 'labels',
+      input_type: 'list',
+      label: 'Labels',
+      description:
+        'Labels are identifiers used by Prefect Agents for selecting flow runs when polling for work. Labels that exist on both the run and the agent will be submitted!'
+    },
+    {
+      arg: 'env',
+      input_type: 'object',
+      label: 'Environment variables',
+      description: 'Additional environment variables to set for the process.'
+    },
+    {
+      arg: 'working_dir',
+      input_type: 'string',
+      label: 'Working directory',
+      description:
+        'The working directory in which to start the process; the directory must already exist. If not provided, will be run in the same directory as the agent was started.'
+    }
+  ]
+}
+
+export const UniversalRun = {
+  type: 'UniversalRun',
+  label: 'Universal',
+  description: 'Run the flow on any agent with matching labels',
+  ref: 'https://docs.prefect.io/api/latest/run_configs.html#universalrun',
+  icon: 'fad fa-globe',
+  args: [
+    {
+      arg: 'labels',
+      input_type: 'list',
+      label: 'Labels',
+      description:
+        'Labels are identifiers used by Prefect Agents for selecting flow runs when polling for work. Labels that exist on both the run and the agent will be submitted!'
+    }
+  ]
+}
+
+export const DockerRun = {
+  type: 'DockerRun',
+  label: 'Docker',
+  description: 'Run the flow as a container on a Docker agent',
+  ref: 'https://docs.prefect.io/api/latest/run_configs.html#dockerrun',
+  icon: 'fab fa-docker',
+  args: [
+    {
+      arg: 'labels',
+      input_type: 'list',
+      label: 'Labels',
+      description:
+        'Labels are identifiers used by Prefect Agents for selecting flow runs when polling for work. Labels that exist on both the run and the agent will be submitted!'
+    },
+    {
+      arg: 'image',
+      input_type: 'string',
+      label: 'Image',
+      description: 'The image to use.'
+    },
+    {
+      arg: 'env',
+      input_type: 'object',
+      label: 'Environment variables',
+      description: 'Additional environment variables to set in the container.'
+    }
+  ]
+}
+
+export const KubernetesRun = {
+  type: 'KubernetesRun',
+  label: 'Kubernetes',
+  description: 'Run the flow as a job on a Kubernetes agent',
+  ref: 'https://docs.prefect.io/api/latest/run_configs.html#kubernetesrun',
+  icon: 'fak fa-kubernetes',
```

Comment:
we probably want to change this to our local font, since we know we can't use the kit info for uploading icons anymore 😞

Reply:
Yeah definitely, I'll tackle that conversion later today >:(